third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -1565,6 +1563,44 @@ func resourceContainerClusterDelete(d *schema.ResourceData, meta interface{}) er
 	return nil
 }
 
+// cleanFailedContainerCluster deletes clusters that failed but were
+// created in an error state. Similar to resourceContainerClusterDelete
+// but implemented in separate function as it doesn't try to lock already
+// locked cluster state, does different error handling, and doesn't do retries.
+func cleanFailedContainerCluster(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
```

Comment:
config, project and location are all defined in the caller, is there a reason to derive them here instead of passing them in?

Reply:
NO it's just uglier and I hate passing multiple strings